LFGBulletinBoard/dungeons/classic.lua

```diff
@@ -16,14 +16,12 @@ if WOW_PROJECT_ID ~= WOW_PROJECT_CLASSIC then return end
 ---@field size number? # The size of the dungeon, only used for raids and battlegrounds
 ---@field isHoliday boolean? # If the dungeon is a holiday event
 
--- Required APIs. Exist in classic even though no Dungeon Finder.
-assert(GetLFGDungeonInfo, tocName .. " requires the API `GetLFGDungeonInfo` for parsing dungeon info")
```

Comment:
No longer using this API as part of the data scraping process for the classic data. Removing all associated code, could be copy pasted back in later if the API is ever needed again